chainercv/links/model/faster_rcnn/faster_rcnn.py

```diff
@@ -112,10 +112,10 @@ def n_class(self):
         # Total number of classes including the background.
         return self.head.n_class
 
-    def __call__(self, x, scale=1.):
+    def __call__(self, x, scales):
```

Comment:
The default value should be set in order to prevent this from becoming `no-compat`